scripts/board_client.py

```diff
@@ -1,13 +1,22 @@
-#!/usr/bin/env python3
+#!/usr/bin/env -S python3 -B
```

Comment:
```suggestion
#!/usr/bin/env python3
```
It seems that not every `env` program supports the `-S` flag